### ISSUE TITLE
Send test result to Windows CMD

### ISSUE THREAD

**Issue by ghost** [gerardroche/sublime-phpunit#92]
Hey there...

Is it possible to pass the results to any Windows CMD (e.g. ConEmu) instead of the default panel in Sublime? Or is this "strategy" not available by default, I presume? And if so, then is it fairly simple to accomplish this?!^_^'

Thanks a lot!

**Comment by gerardroche** [comment]
I'm not familiar with Windows CMD or ConEmu. I'm sure it's possible to implement a as strategy.

**Comment by gerardroche** [comment]
I've added an attempt at supporting a cmd.exe strategy for running tests, but
you can try using the `phpunit.prepend_cmd` setting for custom strategies
(you don't need the dev branch for this, it's available in the latest release).


Using the `phpunit.prepend_cmd` for custom strategies
-----------------------------------------------------

You can try create a custom strategy. When you have them working, let me know
the command structure and I'll add them as concrete strategies. Or you can send
a PR for them if you like too.

**Example:** Custom cmd.exe strategy

Command Palette → Preferences: PHPUnit Settings

```json
{
    "phpunit.prepend_cmd": ["cmd.exe", "/c"]
    "phpunit.strategy": "external"
}
```

The "external" strategy is available since 3.17.1 and prevent the exec output
panel from opening.

If you can test strategies for for things like Powershell and ConEmu, again, I
can add them as concrete strategies, once I know the command structure. I'm just
not in a position to test on Windows.

cmd strategy dev branch
-----------------------

It's in the branch windows-cmd

https://github.com/gerardroche/sublime-phpunit/commit/e6c963724142f8a516a2a7db28e253dd1afd2994

**Example:** Use the cmd.exe strategy

Command Palette → Preferences: PHPUnit Settings

```json
{
    "phpunit.strategy": "cmd"
}
```

**Comment by gerardroche** [comment]
See https://github.com/gerardroche/sublime-phpunit#strategies
